Cargo.toml

```diff
@@ -47,6 +47,7 @@ snappy = ["dep:snap"]
 zlib = ["dep:flate2"]
 lz4 = ["dep:lz4_flex"]
 zstd = ["dep:zstd"]
+wal_disable = []
```

Comment:
Rohan mentioned above:

> Because this option is a bit risky - we wouldn't want it to be set to false by mistake - it's only enabled when a rust feature called wal_disable is enabled.

Basically, just extra paranoia because it's a dangerous setting.

Reply:
I expect a good number of (if not most) users will never want to enable this (though I definitely do for my use case 😄 ), and it's dangerous - so as an extra precaution I just put it behind a feature.